addons/website_payment/static/tests/tours/donation.js

```diff
@@ -68,6 +74,91 @@ registerWebsitePreviewTour('donation_snippet_edition', {
             trigger: ":iframe input#other_amount:not(:checked)",
             run: () => {}, // This is a check
         },
+        {
+            content: "Click on the custom amount radio button",
+            trigger: ":iframe input#other_amount",
+            run: "click"
+        },
+        {
+            content: "Verify the display of a warning message",
+            trigger: ":iframe p#warningMessageId",
```

Comment:
It would be better if we could also display the message of the warning too, what do you think ?

Reply:
I don't think to add message warning. There were many steps to verify; if it fails, then automatically it will display an error. Above this tour, they did the same thing. 
I really don't think to add such verification at each and every step.